### ISSUE TITLE
make sure cors flag also applies to socket

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2923]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.47.5](https://github.com/open-wa/wa-automate-nodejs/compare/4.47.4...4.47.5) (2022-10-21)

- 🐛 fix #2923 [`#2923`](https://github.com/open-wa/wa-automate-nodejs/issues/2923)
- Release 4.47.5 [`dc184d6`](https://github.com/open-wa/wa-automate-nodejs/commit/dc184d6d09d73bb2cb7422aa5139998694cfb9b8)
- 🐛 CLI: `--cors` now on socket server #2923 [`5219692`](https://github.com/open-wa/wa-automate-nodejs/commit/52196922b4446f392f8373fc1050441b257f34b7)
- 🥚 patches update [`913bfea`](https://github.com/open-wa/wa-automate-nodejs/commit/913bfeafe97a24f70d4d8d21030ddd213b511f7b)
- 🥚 patches update [`d68d117`](https://github.com/open-wa/wa-automate-nodejs/commit/d68d11709db24012a6c0f06a410e993272579c5d)
- 🥚 patches update [`76e7a51`](https://github.com/open-wa/wa-automate-nodejs/commit/76e7a5184a3b7110e2cddc11ef1caf0e5f980cbf)
- 🥚 patches update [`e6d407e`](https://github.com/open-wa/wa-automate-nodejs/commit/e6d407e5c3bd244c0f424b8471c64249ed8d8529)
- updated types-only package [`b043722`](https://github.com/open-wa/wa-automate-nodejs/commit/b0437229d6241b76f63676c265f754762596fec5)
